### PR TITLE
*: add del by prefix with e2e tests

### PR DESCRIPTION
**Pull request by gyuho**

### CODE REVIEW

File: e2e/etcdctlv3_test.go [hunk 1]
```diff
@@ -261,6 +240,60 @@ func getTest(cx ctlCtx) {
 	}
 }
 
+func delTest(cx ctlCtx) {
+	defer close(cx.errc)
+
+	var (
+		kvs = []kv{{"key1", "val1"}, {"key2", "val2"}, {"key3", "val3"}}
```
Comment (heyitsanthony) [inline e2e/etcdctlv3_test.go]: `kvs := ...`?

File: e2e/etcdctlv3_test.go [hunk 2]
```diff
@@ -261,6 +240,60 @@ func getTest(cx ctlCtx) {
...
+	)
+	for i := range kvs {
+		if err := ctlV3Put(cx, kvs[i].key, kvs[i].val); err != nil {
+			cx.t.Fatalf("delTest error (%v)", err)
+		}
+	}
+
+	var (
+		keyToGet   = "key"
```
Comment (heyitsanthony) [inline e2e/etcdctlv3_test.go]: `keyToGet := "key"`?

File: e2e/etcdctlv3_test.go [hunk 3]
```diff
@@ -261,6 +240,60 @@ func getTest(cx ctlCtx) {
...
+		deletedNum = 3
+	)
+	if !cx.prefix {
+		keyToGet = "key1"
+		deletedNum = 1
+	}
+
+	if err := ctlV3Del(cx, keyToGet, deletedNum); err != nil {
+		if cx.dialTimeout > 0 && isGRPCTimedout(err) {
```
Comment (heyitsanthony) [inline e2e/etcdctlv3_test.go]: I don't think this is necessary-- Put would have already failed

**Comment by gyuho** [comment]
@heyitsanthony Made them one line for `kvs ...` and delete unnecessary timeout check. PTAL. Thanks.

**Comment by heyitsanthony** [comment]
lgtm

**Pull request merged**